inst/qml/doeResponseSurfaceMethodology.qml

```diff
@@ -26,9 +26,9 @@ Form
         VariablesForm
         {
             AvailableVariablesList { name: "rsmVariablesList" }
-            AssignedVariablesList  { name: "rsmVariables";	        title: qsTr("Predictors");	suggestedColumns: ["scale"]   }
-            AssignedVariablesList  { name: "rsmResponseVariables";	title: qsTr("Response"  );  suggestedColumns: ["scale"]   }
-            AssignedVariablesList  { name: "rsmBlocks";	            title: qsTr("Blocks"  );    suggestedColumns: ["nominal"]   }
+            AssignedVariablesList  { name: "rsmVariables";	        title: qsTr("Predictors (Two)");	suggestedColumns: ["scale"]   }
+            AssignedVariablesList  { name: "rsmResponseVariables";	title: qsTr("Response (One)");  suggestedColumns: ["scale"]   }
+            AssignedVariablesList  { name: "rsmBlocks";	            title: qsTr("Blocks (One)");    suggestedColumns: ["nominal"]   }

```

Comment:
If you want to enforce that people can only add a single variable, you should add `singleVariable: true`.